pgadmin4-17.yaml

```diff
@@ -0,0 +1,164 @@
+package:
+  name: pgadmin4-17
+  version: "8.14"
+  epoch: 0
+  description: "pgAdmin is the most popular and feature rich Open Source administration and development platform for PostgreSQL, the most advanced Open Source database in the world."
+  copyright:
+    - license: PostgreSQL
+  dependencies:
+    provides:
+      - pgadmin4=${{package.full-version}}
+    runtime:
+      - python-${{vars.py-version}}
+      - libpq-${{vars.pg-version}}
+      - nodejs
+
+vars:
+  py-version: 3.13
+  pg-version: 17
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - gcc
+      - glibc-locale-en
+      - krb5-dev
+      - libffi-dev
+      - libjpeg-turbo-dev
+      - libpng-dev
+      - libpq-${{vars.pg-version}}
+      - libtool
+      - make
+      - nasm
+      - nodejs
+      - npm
+      - openssl-dev
+      - pkgconf-dev
+      - postgresql-${{vars.pg-version}}-contrib
+      - postgresql-${{vars.pg-version}}-dev
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}-dev
+      - rust
+      - yarn
+      - yasm
+      - zlib-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: '\.'
+    replace: "_"
+    to: mangled-package-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 9e58717321a434519875773707df78c020bcde40
+      repository: https://github.com/pgadmin-org/pgadmin4
+      tag: REL-${{vars.mangled-package-version}}
+
+  - name: Building Web Assests
+    runs: |
+      cd web
+      export CPPFLAGS="$CPPFLAGS -DPNG_ARM_NEON_OPT=0"
+      npm install -g corepack
+      corepack enable
+      yarn set version berry
+      yarn set version 3
+      yarn install
+      yarn run bundle
+
+  - name: Build package
+    runs: |
+      # Setup Python virtual environment
+      python3 -m venv venv
+      source venv/bin/activate
+
+      # Upgrade basic tools
+      pip install --upgrade pip
+
+      # Install Python dependencies
+      pip install -r requirements.txt
+      pip install sphinx sphinxcontrib-youtube
+      pip install -r web/regression/requirements.txt
+
+      # Build documentation and ignore warnings
+      make -C docs/en_US -f Makefile.sphinx SPHINXOPTS='' html || echo "Documentation warnings ignored"
+
+      # Build Python package
+      make pip
+
+      mkdir -p ${{targets.destdir}}/pgadmin4
+      mkdir -p ${{targets.destdir}}/var/lib/pgadmin
+      mkdir -p ${{targets.destdir}}/var/log/pgadmin
+
+      # Copy in the code and docs
+      cp -R web/ ${{targets.destdir}}/pgadmin4/
+      cp -R docs/ ${{targets.destdir}}/pgadmin4/
+      cp -R venv ${{targets.destdir}}/pgadmin4/
+      cp ./pkg/docker/run_pgadmin.py ${{targets.destdir}}/pgadmin4/
+      cp ./pkg/docker/gunicorn_config.py ${{targets.destdir}}/pgadmin4/
+      cp ./pkg/docker/entrypoint.sh ${{targets.destdir}}/entrypoint.sh
```

Comment:
Is there a particular reason for installing into `/pgadmin4` and `/entrypoint.sh` rather than something like `/usr/share/pgadmin4`, `/usr/bin`?

Reply:
Ah, okay, thanks!